packages/wonder-blocks-testing/src/fixtures/types.js.flow

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
**nit:** I think I can delete this now that I fixed `flowgen`'s object type. I'll check.